runtime/binding/python/py/decode_main.py

```diff
@@ -0,0 +1,26 @@
+import sys
```

Comment:
Movedecoder_main.py to parend direcnionary？
It‘s looks like an example

Reply:
it's a temp test, will refine it in the future.